app/controllers/LoginController.js

```diff
@@ -0,0 +1,63 @@
+var blueprint = require ('@onehilltech/blueprint'),
+    request   = require('superagent'),
+    util      = require ('util')
+  ;
+
+function LoginController () {
+  blueprint.BaseController.call (this);
+}
+
+blueprint.controller (LoginController);
+
+LoginController.prototype.login = function () {
+    var self = this;
+
+    return function (req, res) {
+
+        //preliminary checks
+        if(req.body.name.length < 5){
+            return res.render ('login.pug', {message: "Username not valid"});
+        }
+
+        if(req.body.password.length < 3){
+            return res.render ('login.pug', {message: "Password not valid"});
+        }
+
+        var token;
+
+
+        var userData = {
```

Comment:
We need to update this to not use the hard coded values and use the values from the login text fields.